docker/bybe_dev/docker-compose.yml

```diff
@@ -44,6 +44,8 @@ services:
   redis:
     image: redis:7.2.3
     restart: unless-stopped
+    ports:
```

Comment:
Not directly related to this PR, small adjustement to docker-compose I use for development